### ISSUE TITLE
Editable cells are not allowd to edit some times

### ISSUE THREAD

**Issue by yoursantu** [muhimasri/b-editable-table#39]
Hi,
I am facing a new issue, while editing editable fields, some time it is not allowed to edit input fields. it is throwing console error like: 

```
Error in v-on handler: "TypeError: Cannot set properties of undefined (setting 'isEdit')"
  ```

I am using filter to filter fields based on items category, while applying the filter the editable fields are disabled sometimes. 
please let me know how can i fix this?

Thank you.

![Console Error](https://user-images.githubusercontent.com/14079339/233279797-d96b1d0d-1c9c-4a6b-817d-b492c8f9ad95.png)

**Comment by muhimasri** [comment]
Hello, is the edit cell breaking after filtering rows? If you are not filtering, editing cells working normally?

**Comment by yoursantu** [comment]
Hi,
Yeah, editing cells working normally if i did not apply filter rows...

**Comment by muhimasri** [comment]
Can you please provide me with a sample code of how you are filtering? It will help figure what might be causing the issue.

**Comment by yoursantu** [comment]
Hi,
Sharing the repo for code samples:
https://github.com/yoursantu/be-table

The demo link:
https://ik6z6x-8080.csb.app/

**Comment by muhimasri** [comment]
Thanks for sharing. I'm playing around with the demo and trying to reproduce the issue.

Basically, when filtering a category, the editable fields are disabled sometimes? Is there a specific scenario when this happens, or it's just random?

**Comment by yoursantu** [comment]
Hi,
Thank you for your reply,
Yeah, the editable fields are randomly disabled (not every time) when filtering the category.  And once the editable fields are disabled after filtering the category will be worked again after clearing the filter.

Thank you.

**Comment by muhimasri** [comment]
Hi there, I wasn't able to reproduce the specific issue you're having, however I found a different bug related to filtering and editing the wrong cells. Not sure if that will fix your issue but please install latest release "1.0.2" and let me know how it goes.

**Comment by yoursantu** [comment]
Hi,
Thank you very much for your response and your valuable time!!
I have updated the package to the latest and tried. 
But my issue still exists, and now it showing different console error than earlier... 

![Screenshot from 2023-05-29 22-33-19](https://github.com/muhimasri/b-editable-table/assets/14079339/8033b991-4a98-41da-a10e-03f409f4f0cb)

**Comment by muhimasri** [comment]
No problem! I will investigate and try to figure the issue.

**Comment by muhimasri** [comment]
Hi there, there is a new release v1.0.3, it has some new fixes. Please install and let me know if your issue still persists.

**Comment by yoursantu** [comment]
Hi,
After testing it for a long time, i confirm that the issue has been resolved in the latest version v1.0.3! (**Editable fields disabled** sometimes on filter applied)

Thank you very much for all your effort.

**Comment by yoursantu** [comment]
Hi,
After testing it for a long time, i confirm that the issue has been resolved in the latest version v1.0.3! (**Editable fields disabled** sometimes on filter applied)

Thank you very much for all your effort.
